astropy/constants/__init__.py

```diff
@@ -23,11 +23,11 @@
 
 del units
 
-from . import cgs  # noqa
-from . import si  # noqa
-from . import utils as _utils  # noqa
-from .config import codata, iaudata  # noqa
-from .constant import Constant, EMConstant  # noqa
+from . import cgs  # noqa: F401, E402
+from . import si  # noqa: F401, E402
+from . import utils as _utils  # noqa: F401, E402
+from .config import codata, iaudata  # noqa: F401, E402
+from .constant import Constant, EMConstant  # noqa: F401, E402
```

Comment:
I'm curious if there would be a better way to deal with the warnings in ``__init__.py`` files since this is so common. There's nothing to fix here, so I wonder if we can somehow run flake8 in a way that it would automatically ignore F401/E402 when looking at ``__init__.py`` files?